generate-cpio-rootfs.sh

```diff
@@ -292,6 +299,8 @@ case $1 in
         ;;
      "hikey")
         ;;
+    "hikey960")
+        ;;
```

Comment:
This whole "Extra stuff" section is useless since no platform does anything here. I suggest adding a patch "Remove useless case statement in generate-cpio-rootfs.sh" or something like that, prior to this one.